bundles/org.openhab.voice.voskstt/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -5,7 +5,8 @@
 
 	<type>voice</type>
 	<name>Vosk Speech-to-Text</name>
-	<description>osk STT Service uses vosk-api to perform offline speech-to-text in openHAB.</description>
+	<description>Vosk STT Service uses vosk-api to perform offline speech-to-text in openHAB.</description>
+	<connection>local</connection>
```

Comment:
I think that connection can be none, Vosk library is included in the bundle and executed locally by the add-on.